test/unit/specs/dojos/service.spec.js

```diff
@@ -24,7 +24,6 @@ describe('Dojos Service', () => {
   const expectedResult = { name: 'Cool Dojo' };
 
   const expectedDojos = [{
```

Comment:
If they do, the test fails because the processed snake case dojos don't have it and they are expected to equal expectedDojos